_sass/components/homepage/_infocols.scss

```diff
@@ -0,0 +1,92 @@
+.infocols-container {
+  max-width: $container-max-width;
+  margin: auto;
+}
+
+.infocols-section {
+  flex-direction: column;
+  align-items: center;
+  text-align: center;
+  gap: 3rem;
+  flex-wrap: wrap;
+
+  .infocols-header {
+    flex-direction: column;
+    gap: 1rem;
+  }
+
+  .infocols-footer {
+    flex-direction: row;
+    gap: 0;
+
+    .infocols-linktext {
+      font-size: 1.125rem;
+      text-transform: uppercase;
+      transform: translateY(
+        -0.09rem
+      ); // making it centered with the arrow-right icon
+    }
+  }
+}
+
+.infocols-infobox-section-grid {
+  flex-direction: column;
+  flex: 1;
+  gap: 1rem;
+}
+
+.infocols-infobox-section.space-between {
+  justify-content: space-between;
+}
+
+.infocols-infobox-section {
+  align-items: flex-start;
+  justify-content: center;
+  text-align: left;
+  gap: 1rem;
+  flex-wrap: wrap;
+  flex-direction: column;
+  width: 100%;
+
+  // md: Make it into row for tablet onwards
+  @media screen and (min-width: 769px) {
```

Comment:
nit: prefer using `map-get($breakpoints, "md")` (maybe wrong size, please check)

Reply:
this allows us to use the same break point on preview also (and scale it to account for drawer) to also properly have correct breakpoints

Reply:
@harishv7 did it work for a live site? I vaguely recall this can't work but if it does then that would be great!

Reply:
@dcshzj it works on staging

Reply:
`map.get` doesn't work but `map-get` does for reasons unknown...

Reply:
Very interesting, TIL! At least now we can properly use the breakpoints. Probably should do this for text cards too so we can standardise the components, but let's do that in a separate PR.

Reply:
Oi dun ot leh knncv